package.json

```diff
@@ -33,6 +33,9 @@
     "db:migrate:prod": "knex migrate:latest --verbose --env production",
     "db:seed": "knex seed:run"
   },
+  "engines": {
+    "node": ">= 14.0.0"
```

Comment:
This is required for the `throwIfNoEntry` configuration option used in the new call:

```js
fs.statSync(seedDir, { throwIfNoEntry: false })
```